cltk/stop/tamil/stops.py

```diff
@@ -0,0 +1,133 @@
+"""This stopword list was compiled by SeenivasanSeeni
+"""
+
+__author__ = ['SeenivasanSeeni <seeni0424@gmail.com>']
+__license__ = 'GPL License.'
+
+#main source of stop words is in https://github.com/AshokR/TamilNLP/blob/master/Resources/TamilStopWords.txt
```

Comment:
Can you please move this comment to the top along with [1](https://github.com/cltk/cltk/pull/848/files#diff-0e1bc437132886d43f7e8a222a13cdb0R1) .